unic/Cargo.toml

```diff
@@ -27,7 +28,7 @@ unic-emoji = { path = "emoji/", version = "0.9.0" }
 unic-idna = { path = "idna/", version = "0.9.0" }
 unic-normal = { path = "normal/", version = "0.9.0" }
 unic-segment = { path = "segment/", version = "0.9.0" }
-unic-ucd = { path = "ucd/", version = "0.9.0" }
+unic-ucd = { path = "ucd/", version = "0.9.0", features = [] }
```

Comment:
Not sure if this is the best way to do it, the indention is to make the `unihan` feature turned-off by default.
Currently the `unic-ucd` only have a `unihan` feature, so this is probably fine, if we start to add more features to `unic-ucd`, then we will need to explicit specify the features here if we want the feature to be opt-in by default.

Reply:
I think it's better to have `default = [ <enabled-features> ]` on the `unic-ucd` side. If there's no other enabled-by-default feature there, I think we can rely on the default value for `default = []` then.

So, just create a `[features]` item for unihan and don't put `unihan` in the list of deps?

Reply:
Right, `unic-ucd = { path = "ucd/", version = "0.9.0" }` should work, I've removed `features = []`.

I've verified with an example project that `cargo build` won't build `unic-ucd-unihan`, and `cargo build --feature unihan` will. Running `cargo build` in `./unic` or `./unic/ucd` works in the same way.

I thought it was not working because when I ran `cargo build` in the root of the project it built `unic-ucd-unihan` (any ideas why?).

Reply:
Since what we have is a "virtual workspace" (that is, the root `Cargo.toml` is has a `[workspace]` table and no `[package]` table) `cargo build` in the root directory defaults to `cargo build --all`.

Reply:
Ah, had a read through https://doc.rust-lang.org/cargo/reference/manifest.html#virtual-manifest now it makes more sense, thanks!